### PR TITLE
feat(contribute): add CONTRIBUTING

### PR DESCRIPTION
**Pull request by GuillaumeOj**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Contributing to react-crisp
+
+First off, thanks for taking the time to contribute!
+
+The following as a set of guidelines for contributing to react-crisp.
```
Comment (jd) [inline CONTRIBUTING.md]:
```suggestion
The following is a set of guidelines for contributing to react-crisp.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+These are mostly guidelines, not rules. Use your best judgment, and fell free
```
Comment (jd) [inline CONTRIBUTING.md]:
```suggestion
These are mostly guidelines, not rules. Use your best judgment, and feel free
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[react-crisp Code of Conduct](CODE_OF_CONDUCT.md). By participating, you
```
Comment (jd) [inline CONTRIBUTING.md:12]: is there a COC?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+are expected to uphold this code. Please report unacceptable behavior to
+[jd@mergify.io](mailto:jd@mergify.io).
```
Comment (jd) [inline CONTRIBUTING.md]: Might be better to use a non-personal email.
I've created `opensource@mergify.io`

**Review by jd — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (GuillaumeOj) [reply CONTRIBUTING.md:12]: Yes just right here :innocent: https://github.com/Mergifyio/react-crisp/pull/9

**Review by jd — APPROVED** [review]

**Review by sileht — APPROVED** [review]

**Pull request merged**